dotCMS/src/integration-test/java/com/dotcms/enterprise/publishing/remote/PushPublishBundleGeneratorTest.java

```diff
@@ -597,5 +605,80 @@ public void Test_GenerateBundle_PageWithTemplateDeleted_PageSuccessfullyAddedToB
 
     }
 
+    /**
+     * Given Scenario: If a file asset has zero files and CONTENT_ALLOW_ZERO_LENGTH_FILES set to true,
+     *                  we should be able to generate the bundle without issues.
+     * ExpectedResult: The bundle should be generated without issues and the page added to the bundle.
+     */
+    @Test
+    public void Test_GenerateBundle_FileAssetZeroBytes_ContentAllowZeroLengthFilesSetToTrue_Success() throws Exception {
+        final boolean allowZeroLengthFilesDefault = Config.getBooleanProperty("CONTENT_ALLOW_ZERO_LENGTH_FILES", true);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "CONTENT_ALLOW_ZERO_LENGTH_FILES" appears 6 times in this file; the first occurrence is on line 615](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5701517)